src/Compute/Compute/help/New-AzVmss.md

```diff
@@ -350,6 +350,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -HostGroupId
+Specifies the dedicated host group the virtual machine scale set will reside in.
+
+```yaml
+Type: System.String
+Parameter Sets: SimpleParameterSet
+Aliases: HostGroup
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
```

Comment:
The src/Compute/Compute/help/New-AzVM.md file shows the Accept Pipeline Input is True, I think these should match. Which is preferred? 
I don't see why accepting from pipeline shouldn't be True.

Reply:
Why not :)?